DSESwift/DataPoint.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  DataPoint.swift
+//  SensePlatform
+//
+//  Created by Tatsuya Kaneko on 16/09/15.
+//
+//
+
+import Foundation
+
+class DataPoint{
+    var sensorId = -1
+    private(set) var value = ""
+    var date = NSDate()
+    var synced = false
+
+    init(sensorId: Int, value: String, date: NSDate, synced: Bool) {
+        self.sensorId = sensorId
+        self.value = value
```

Comment:
@kanekin in Java we currently also store the type of value in a separate field, so you can correctly restore a number as either float, int, etc. That sort of type information is lost when stringifying a numeric value. I'm not sure if that is actually needed in the future usage of these DataPoints, you should be able to get this information too from the sensor class.

@excaliburnan what do you think?